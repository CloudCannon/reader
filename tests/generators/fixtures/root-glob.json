{
	"source": ".",
	"collections_config": {
		"everything": {
			"path": ".",
			"glob": [
				"tests/generators/fixtures/standard/**/nested/*"
			],
			"output": true,
			"url": "hello/{title|slugify}"
		}
	},
	"_comments": {},
	"_options": {},
	"_structures": {},
	"_select_data": {},
	"generator": {},
	"source_editor": {},
	"paths": {
		"uploads": "assets/uploads"
	},
	"base_url": ""
}