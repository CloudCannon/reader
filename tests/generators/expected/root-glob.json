{
  "source": ".",
  "collections_config": {
    "everything": {
      "path": ".",
      "glob": [
        "tests/generators/fixtures/standard/**/nested/*"
      ],
      "output": true,
      "url": "hello/{title|slugify}"
    }
  },
  "_comments": {},
  "_options": {},
  "_structures": {},
  "_select_data": {},
  "generator": {},
  "source_editor": {},
  "paths": {
    "uploads": "assets/uploads"
  },
  "base_url": "",
  "time": "2000-11-22T00:00:00.000Z",
  "cloudcannon": {
    "name": "cloudcannon-reader",
    "version": "0.0.1"
  },
  "version": "0.0.3",
  "data": {},
  "collections": {
    "everything": [
      {
        "date": "2020-07-22T00:00:00.000Z",
        "title": "Egg",
        "categories": [
          "tips"
        ],
        "author_staff_member": "betty",
        "path": "tests/generators/fixtures/standard/_posts/nested/2020-07-22-egg.md",
        "collection": "everything",
        "url": "hello/egg"
      }
    ]
  }
}